src/components/circular-progress/circular-progress.tsx

```diff
@@ -0,0 +1,82 @@
+import { Component, h, Prop } from '@stencil/core';
+import { CircularProgressSize } from './circular-progress.types';
+// import { CircularProgressSize } from '@limetech/lime-elements'; // Kia: I'd like something like this, but it won't work :/
+
+/**
+ * The circular progress component can be used to visualize the curent state of
+ * a progress in a scale; for example percentage of completion of a task.
+ *
+ * Its compact UI makes the component suitable when there is not enough screen
+ * space available to visualise such information.
+ *
+ * This component allows you to define your scale, from `0` to a desired
+ * `maxValue`; and also lets you chose a proper `suffix` for your scale.
+ *
+ *
+ * @exampleComponent limel-example-circular-progress
+ * @exampleComponent limel-example-circular-progress-sizes
+ * @exampleComponent limel-example-circular-progress-props
+ * @exampleComponent limel-example-circular-progress-css-variables
+ * @exampleComponent limel-example-circular-progress-percentage-colors
+ */
+@Component({
+    tag: 'limel-circular-progress',
+    shadow: true,
+    styleUrl: 'circular-progress.scss',
+})
+export class CircularProgress {
+    /**
+     * The value of the progress bar. Should be between `0` and `maxValue`.
+     */
+    @Prop()
+    public value: number = 0;
+
+    /**
+     * The maximum value within the scale that the progress bar should visualize. Defaults to `100`.
+     */
+    @Prop()
+    public maxValue: number = 100; // eslint-disable-line no-magic-numbers
+
+    /**
+     * The suffix which is displayed after the `value`, must be one or two characters long. Defaults to `%`
+     */
+    @Prop()
+    public suffix: string = '%';
+
+    /**
+     * When set to `true`, makes the filled section showing the percentage colorful. Colors change with intervals of 10%.
+     */
+    @Prop()
+    public displayPercentageColors: boolean = false;
+
+    /**
+     * Determines the visual size of the visualization from a preset size. This property can override the `--circular-progress-size` variable if it is specified.
+     */
+    @Prop({ reflect: true })
+    public size: CircularProgressSize;
+
+    public render() {
+        const classList = {
+            'lime-circular-progress': true,
+            'displays-percentage-colors': this.displayPercentageColors,
+        };
+
+        const currentPercentage = (this.value * 100) / this.maxValue + '%'; // eslint-disable-line no-magic-numbers
+
```

Comment:
declare this 
`const value = (Math.round(this.value * 10)/10);` 

This will round it to a max of 1 decimal place. For example `6.28` becomes `6.3`

Then use this ` {abbreviate(value)}` on line 77 instead. This will shorten large number for example 
`1234` becomes `1k`

Reply:
awesome!